### PR TITLE
Removed ui_old folder

### LINKED ISSUE #120: Do we still need ui_old?
It bugs me every time I see the build script touch this directory. Is it actually necessary?

https://github.com/starteam/starcellbio_html/tree/develop/html_app/ui_old

### PR DESCRIPTION
**Pull request by annagav**
Fixes #120.

### CODE REVIEW

**Comment by carsongee** [comment]
This needs to be rebased and squashed down to one commit

File: html_app/ui/MainFrame.js [hunk 1]
```diff
@@ -996,118 +991,6 @@ scb.ui.MainFrame = function scb_ui_MainFrame(master_model, context) {
         self.show(state);
     })();
 
-    // init is really not used any more I'll need to move on...
-    self.init = function () {
```
Comment (carsongee) [inline html_app/ui/MainFrame.js]: Is this strictly required for removing ui_old?  I love deleting code, but if this isn't a direct dependency, it should probably be a different PR so we can revert one without the other if we end up needing it.

File: static_assets.yml [hunk 2]
```diff
@@ -14,7 +14,6 @@ javascripts:
   - instructor/gen/instructor_facs_page2.soy.js
   - js/jquery-ui.js
   - ui/WesternBlotGelView.js
-  - ui_old/WorkspaceView.js
```
Comment (carsongee) [inline static_assets.yml:17]: yay, removing assets :ghost:

**Comment by carsongee** [comment]
This looks good to me after squash and rebase :kimono:

**Comment by annagav** [comment]
Done!

**Comment by carsongee** [comment]
Just curious, where was the conflict?

**Comment by annagav** [comment]
It was in static_assets file, coin_slider file happen to be next to ui_old file.
